scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -213,8 +216,12 @@ Function AppendInstallationOption([string] $testCommand)
 Function CleanUp
 {
     PrintHighlightedMessage "Test Clean Up"
-    Write-Host "Do IoT Edge Moby system prune"
 
+    Write-Host "Uninstall iotedged"
+    Invoke-Expression $InstallationScriptPath
+    Uninstall-IoTEdge -Force
```

Comment:
Do you know Uninstall-IoTEdge will uninstall moby as well? if so, we can remove "docker system prune" below.

Reply:
I tried _Uninstall-IoTEdge_ and it does uninstall _moby_ as well.